docs/faq/proof-of-stake/general.md

```diff
@@ -132,59 +115,56 @@ Currently, a platform for the community to submit and advocate a voting agenda i
 
 ---
 
-#### 11. Is PoS susceptible to large exchanges using their customers’ decred? 
+#### 11. Is Proof-of-Stake (PoS) susceptible to large exchanges using their customers’ DCR? 
 
-The number of decred a person (or exchange) has means nothing to PoS. It's the number of tickets you have. Funds used to purchase tickets are locked until the ticket they purchased votes. This means that
-decred involved in PoS are effectively nontransferable. For an exchange to use their customers’ decred for voting, they would have to transfer them out of the wallets and lock them for up to 5 months. People
-would notice their balances change (decred locked in PoS will not show as spendable) and they would not be able to withdraw any funds so the exchange would suffer a large loss of liquidity.
+The amount of DCR a person (or exchange) posesses doesn't matter, only the number of tickets. Funds used to purchase tickets are locked until the ticket they purchased votes. This means that DCR involved in PoS are effectively nontransferable. For an exchange to use their customers’ DCR for voting, they would have to transfer them out of the wallets and lock them for up to 5 months. People would notice their balances change (DCR locked in PoS will not show as spendable) and they would not be able to withdraw any funds so the exchange would suffer a large loss of liquidity.
 
-Further there is a hard limit of 20 tickets added per block so no exchange could flood the pool faster than this.
+Furthermore, there is a hard limit of 20 tickets added per block, so no exchange could flood the pool faster than this.
 
-Finally, there's a soft cap on the total number of tickets in the pool. Every 144 blocks (2880 tickets) the ticket price is adjusted based on the number of tickets in the pool and the rate that new tickets
-were added in the last window. Eventually the ticket price would be so high that even an exchange wouldn't be able to buy many tickets. And remember that even if they did that their DCR are locked so they
-can't buy more when the price drops again.
+Finally, there's a soft cap on the total number of tickets in the pool. Every 144 blocks (2880 tickets) the ticket price is adjusted based on the number of tickets in the pool and the rate that new tickets were added in the last window. Eventually the ticket price would be so high that even an exchange wouldn't be able to buy many tickets. And remember that even if they did that their DCR are locked so they can't buy more when the price drops again.
 
 ---
 
-#### 12. Is PoS susceptible to influence from large balance holders such as the original developers? 
+#### 12. Is Proof-of-Stake (PoS) susceptible to influence from large balance holders such as the original developers? 
+
+Some of the figures below are outdated, based on the ticket pricing algorithm before [DCP-0001](https://github.com/decred/dcps/blob/master/dcp-0001/dcp-0001.mediawiki) was voted in and activated.
```

Comment:
Could you remove this disclaimer please and open a new issue for these outdated numbers.

Reply:
Disclaimer removed. Created Issue: https://github.com/decred/dcrdocs/issues/712